SourceryFramework/Sources/Parsing/FileParser.swift

```diff
@@ -357,8 +357,11 @@ extension FileParser {
 // MARK: - Variables
 extension FileParser {
 
-    private func inferType(from string: String) -> String? {
-        let string = string.trimmingCharacters(in: .whitespaces)
+    private func inferType(from input: String) -> String? {
+        let string = input
+            .trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
To be improved at some point by a custom function implementation that would do both phases at once, but good enough for now